modules/nw-networkpolicy-multitenant-isolation.adoc

```diff
@@ -12,7 +16,14 @@ project namespaces.
 
 .Prerequisites
 
-* Your cluster is using a default CNI network provider that supports `NetworkPolicy` objects, such as the OpenShift SDN network provider with `mode: NetworkPolicy` set. This mode is the default for OpenShift SDN.
+* Your cluster is using a cluster network provider that supports `NetworkPolicy` objects, such as
+ifndef::ovn[]
+the OpenShift SDN network provider with `mode: NetworkPolicy` set.
+endif::ovn[]
+ifdef::ovn[]
+the OVN-Kubernetes network provider or the OpenShift SDN network provider with `mode: NetworkPolicy` set.
```

Comment:
I don't see a preview of this version of the prereq item, but seems ok with the `ifdef::ovn[]` here. 🤷

Reply:
@bobfuru, yeah, I need to fake it locally by defining `:product-version` manually.